buildSrc/src/main/groovy/io.micronaut.build.internal.test-application-hibernate.gradle

```diff
@@ -0,0 +1,8 @@
+plugins {
```

Comment:
Instead having hibernate5 or hibernate6 in prev version, I think it makes sense have just hibernate since we support only current version of hibernate (this applies to tests etc)